bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -819,6 +829,14 @@ impl<'a> CircuitInputStateRef<'a> {
             self.handle_reversion();
         }
 
+        // If current call has caller.
+        // TODO: use is_root??
```

Comment:
Is it still unsure how to best check this case?

Reply:
No, only a legacy comment, removed.